src/main/java/uk/gov/ida/verifyserviceprovider/exceptions/MissingUnsignedAssertionsHandlerException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.ida.verifyserviceprovider.exceptions;
+
+public class MissingUnsignedAssertionsHandlerException extends RuntimeException {
+    public MissingUnsignedAssertionsHandlerException() {
+        super("UnsignedAssertionsHandler missing: UnsignedAssertinosHandler can not be null in eIDAS enabled VSP");
```

Comment:
There's a typo here.

But I feel like this exception should not exist: surely we have a standard exception we can throw for bad configuration. This should not happen IRL, right? And I don't necessarily think it merits a test. either, surely we don't test that all of our code that expects a dependency to be injected haven't had `null` injected by mistake?

Reply:
I had a look in the currently defined exceptions and there wasn't anything suitable, hence this. And I agree about not testing all our code that expects a dependency, but this is specifically about a a dependency that may or may not be null depending on a particular scenario.